python/servo/build_commands.py

```diff
@@ -701,6 +705,10 @@ def package_generated_shared_libraries(libs, build_path, servo_exe_dir):
                 if not package_msvc_dlls(servo_exe_dir, target_triple, vcinstalldir, vs_version):
                     status = 1
 
+                # UWP build hololens
+                if uwp:
+                    build_uwp_hololens(target_triple, dev)
```

Comment:
We will want to check the return value here and set status to 1 if the build process fails.